lib/components/Charts/CategoryBarChart/index.tsx

```diff
@@ -0,0 +1,113 @@
+import {
+  Tooltip,
+  TooltipContent,
+  TooltipProvider,
+  TooltipTrigger,
+} from "@/ui/tooltip"
+import { autoColor } from "../utils/colors"
+
+export interface CategoryBarProps {
+  data: {
+    name: string
+    value: number
+    color?: string
+    muted?: boolean
+  }[]
+  legend: boolean
+}
+
+export function CategoryBar({ data, legend = true }: CategoryBarProps) {
+  const total = data.reduce((sum, category) => sum + category.value, 0)
+
+  return (
+    <TooltipProvider>
+      <div className="w-full">
+        <div className="flex h-2 gap-1 overflow-hidden">
+          {data.map((category, index) => {
+            const percentage = (category.value / total) * 100
+            const color = category.color || autoColor(index)
+
+            const formatPercentage = (value: number): string => {
+              const percentage = (value / total) * 100
+              return percentage % 1 === 0
+                ? percentage.toFixed(0)
+                : percentage.toFixed(1)
+            }
+
+            return (
+              <Tooltip key={category.name}>
+                <TooltipTrigger
+                  className="h-full cursor-default overflow-hidden rounded-[2px]"
```

Comment:
I think we are avoiding the custom values like rounded-[2px], but I understand it's the only way